### ISSUE TITLE
Why Python instead of LC?

### ISSUE THREAD

**Issue by zascas** [trevordevore/livecode-sublimetext#17]
Hi, out of curiosity.

You choose Python because is better suite to parser text than LC or just because it simplify the Package installation on Sublime?.

Thanks.

**Comment by trevordevore** [comment]
It is because Sublime Text uses Python. LiveCode isn’t an option.

**Comment by zascas** [comment]
I have been reading a few in your syntax files for LC and LCB, also checking TextMate syntax files it look like a Lex definitions of tokens, I found Sublime Text a bit expensive and I will try to learn form your package a bit more around this topic.

I would like to define domain - specific languages, fo the end user to manipulate the domain specific information inside the applications, just a few words that have very specific meaning for his experimenting activity.

Thanks for your kind answer  for an obvious question, on my reading around you package I found out the  interface of Sublime text is Python based,  just as you said LC wasn't an option.

All the best and take kare.
